src/main/java/com/deavensoft/paketomat/center/CenterController.java

```diff
@@ -37,8 +45,22 @@ public List<Package> getAllPackages()
     @ApiResponse(responseCode = "200", description = "New package added")
     public int savePackage(@RequestBody Package newPackage)
     {
+        newPackage.setStatus(Status.NEW);
         centerServiceImpl.save(newPackage);
         log.info("New package added to the database");
+
+        User user = userServiceImpl.findUserById(newPackage.getReciever()).get();
+        if (user == null) {
```

Comment:
Optional value can hold either a value or not. The value held in the Optional can be accessed using the get() method, but it will throw a NoSuchElementException if there is no value present. To avoid the exception, calling the isPresent() or ! isEmpty() method should always be done before any call to get().